<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>181</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>201</width>
    <height>181</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>201</width>
    <height>181</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите число:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
